doc/reverse_engineering/networking/13-other.md

```diff
@@ -0,0 +1,351 @@
+# Others
+
+## Resign/Disconnect
+
+Used for resigning and disconnecting.
+
+### Definition
+
+```ruby
+def Resign
+  int8 :action_identifier
+  int8 :player_number
+  int8 :player_id
+  int32 :disconnect
+end
+```
+
+### Description
+
+*:action_identifier*  
+Always has the value `0x0b`.
+
+*:player_number*  
+The player's number which can be changed in the lobby. Is only different from *:player_id* if playing in coop mode.
+
+*:player_id*  
+The ID of the player.
+
+*:disconnect*  
+Determines whether the player gave up himself (`0x00`) or was disconnected (`0x01`).
+
+### Examples
+
+`0b 02 03 00`
+
+>`0b` &mdash; action_identifier  
+>`02` &mdash; player_number  
+>`03` &mdash; player_id  
+>`00` &mdash; disconnect
+
+## Save game
+
+Used for saving the game.
+
+### Definition
+
+```ruby
+def Save
+  int8 :action_identifier
+  int8 :exit
+  int8 :player_id
+  unknown :data
+end
+```
+
+### Description
+
+*:action_identifier*  
+Always has the value `0x1b`.
+
+*:exit*  
+Represents whether this action is a normal Save (`0x00`) or a Save & Exit (`0x01`).
+
+*:player_id*  
+The ID of the player saving the game.
+
+*:data*  
+A 253 byte long data field which cannot be parsed easily. Contains filename and presumably a checksum.
```

Comment:
This is only the filename and the checksum. The checksum is 4 bytes at the very end. Anything in between the filename and the checksum is uninitialized memory.

Reply:
Uuuh, so it really is that simple. Fixed it.